Assets/HoloToolkit/Input/GazeManager.cs

```diff
@@ -62,6 +63,7 @@ private void UpdateRaycast()
                            MaxGazeDistance,
                            RaycastLayerMask);
 
+            GameObject oldHitObject = currentHitObject;
```

Comment:
NIT: Please use similar naming convention as in GestureManager.cs.
oldHitObject to be oldFocusedObject
currentHitObject to focusedObject